Django_Blog/09-Update-User-Profile/django_project/users/models.py

```diff
@@ -10,12 +10,12 @@ class Profile(models.Model):
     def __str__(self):
         return f'{self.user.username} Profile'
 
-    def save(self):
-        super().save()
+    def save(self, *args, **kwargs):
+        super(Profile, self).save(*args, **kwargs)
 
         img = Image.open(self.image.path)
 
         if img.height > 300 or img.width > 300:
-            output_size = (300, 300)
+            output_size = (300,300)
             img.thumbnail(output_size)
             img.save(self.image.path)
```

Comment:
no

Reply:
What does no means?

Reply:
If we enter an already registered usernme it showing an error again

Reply:
For that you create a check if already exist showError()